vpt/sinks/file_store/__init__.py

```diff
@@ -0,0 +1,47 @@
+import os
+import contextlib
+from pathlib import Path
+from typing import Union
+
+import keyboard
+import mouse
+import numpy as np
+import soundfile as sf
+
+from vpt.sinks.base import SinkBase
+from vpt.sources.base import SourceBase
+
+
+class FileStore(SinkBase):
+    def __init__(self, dir_name: str, mouse_source: SourceBase,
+                 keyboard_source: SourceBase, audio_source: SourceBase):
+        super().__init__()
+        self.dir = Path(dir_name)
+
+        self.mouse_file = open(self.dir / 'mouse_output.txt', 'w+')
+        mouse_source.get_data_stream().subscribe(self._process_mouse)
+
+        self.keyboard_file = open(self.dir / 'keyboard_output.txt', 'w+')
+        keyboard_source.get_data_stream().subscribe(self._process_key)
+
+        with contextlib.suppress(FileNotFoundError):
+            os.remove(self.dir / 'audio.wav')
+        self.audio_file = sf.SoundFile(self.dir / 'audio.wav',
+                                       mode='w', samplerate=44100, channels=1)
+        audio_source.get_data_stream().subscribe(self._process_audio)
+
+    def _process_audio(self, frame: np.ndarray):
```

Comment:
We defined a different name for these in the diagram. Should we update the diagram?

Reply:
I don't think it matters a lot, since they are supposed to be private methods. Just simply remove them from the diagram (or update the names and mark them as private)

Reply:
It's just that in SQLite store, I've made them public, and if we were to remove them from the diagram, what would there be left?